kzz7yh-a11710/cod-ve07v1_kzz7yh-a11710.xml

```diff
@@ -95,8 +96,8 @@
       <lb ed="#V" n="30"/>illud quod est in anima actualissimum: hobmeni autem sunt actus. 
     </p>
                <p xml:id="kzz7yh-a11710-d1e168">
-                  <lb ed="#V" n="31"/>Contra Imago semper manet in anima. Unde 
-      priam<lb ed="#V" n="32" break="no"/>In imagine pertransit homo. Sed actus 
+                  <lb ed="#V" n="31"/>Contra Imago semper manet in anima. Unde partem
+      <lb ed="#V" n="32" break="no"/>In imagine pertransit homo. Sed actus 
```

Comment:
ps with line over is often "psalmus" or "psalmo" depending on context. 

Again if words get separated we need to remove the break="no"

```suggestion
                  <lb ed="#V" n="31"/>Contra Imago semper manet in anima. Unde psalmus
      <lb ed="#V" n="32"/>In imagine pertransit homo. Sed actus 
```